.github/workflows/rust.yml

```diff
@@ -94,6 +94,46 @@ jobs:
         run: cargo test -vv --features static-link-z3
       - name: Test `z3` with statically linked Z3 and `arbitrary-size-numeral` enabled
         run: cargo test --manifest-path z3/Cargo.toml -vv --features 'static-link-z3 arbitrary-size-numeral'
+  build_with_vcpkg_installed_z3:
+    strategy:
+      matrix:
+        build: [linux, macos, windows]
```

Comment:
For now, let's just do this on Windows since that's where GitHub has it installed.

Reply:
I saw it failing on Linux and that caused the macOS and Windows builds to get cancelled. (At least in the run that I looked at.)

Reply:
I've added an option to disable fail-fast in my branch. Here's the [result](https://github.com/TheVeryDarkness/z3.rs/actions/runs/6561354966/job/17820867395). I guess the output implies that the ABI doesn't match.

Reply:
This may be fixed by cleaning the vcpkg build tree.